### DISCUSSION TITLE
中文文案排版指北（简体中文版） — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#213] [Announcements]
https://mazhuang.org/wiki/chinese-copywriting-guidelines/

**Comment by kazuhaxy** [comment]
数字与单位之间是否加空格应该延续该单位与数字在全英文书写环境下是否加空格，文中所举的“正确”的例子「我家的光纤入户宽带有 10 Gbps，SSD 一共有 20 TB。」恰恰是不合适的。

具体例子参考：
https://www.apple.com/imac-pro/
https://www.apple.com/cn/imac-pro/
https://www.microsoft.com/zh-cn/surface/devices/surface-pro/overview

**Comment by mzlogin** [comment]
经你这么一说，对比了两种写法，`20TB` 貌似还真更符合我的审美。

> 数字与单位之间是否加空格应该延续该单位与数字在全英文书写环境下是否加空格。

这个说法挺明了的，请问有出处吗？

有点遗憾的是苹果的中英文本来应该作为好的范本，但它们一致性做得并不好，中文网站发现了 `4.5 GHz` 和 `400 GB/s` 的写法。

**Comment by finch001** [comment]
我个人感觉数字与单位之间不加空格反而更舒适，『我家的光纤入户宽带有 10Gbps，SSD 一共有 20TB。』

**Comment by uvim** [comment]
> @kazuhaxy
> 数字与单位之间是否加空格应该延续该单位与数字在全英文书写环境下是否加空格，文中所举的“正确”的例子「我家的光纤入户宽带有 10 Gbps，SSD 一共有 20 TB。」恰恰是不合适的。
> 
> 具体例子参考：
> https://www.apple.com/imac-pro/
> https://www.apple.com/cn/imac-pro/
> https://www.microsoft.com/zh-cn/surface/devices/surface-pro/overview

特地去看了下，确实苹果使用的也是数字和单位不加空格的风格，如：
> 8GB 或 16GB 高带宽显存  
> 400GB/s 显存带宽

同意单位和数字之间不加空格的审美观。^_^

**Comment by mzlogin** [comment]
@finch001 
@uvim 

确实如此，我后来修改一下推荐用法😄 ~

**Comment by xys2015** [comment]
我想默默的请问一下，您说苹果中国，微软中国都遵守了"中文文案排版指北"，请问有何凭据吗？

**Comment by mzlogin** [comment]
> @xys2015
> 我想默默的请问一下，您说苹果中国，微软中国都遵守了"中文文案排版指北"，请问有何凭据吗？

这个是这样的：本文档的「谁在这样做？」一节里列出的网站，并非是指它们遵守本文档，而是说它们采用的排版风格上与本文罗列的一些规则是重合度比较高的。它们极可能在本文档出现之前就采用了这样的风格。

**Comment by yujiangshui** [comment]
日期这个有建议吗？比如：“GitHub 免费是在2019年1月4日左右的时候” 要怎么空？我是：“GitHub 免费是在 2019 年 1 月 4 日左右的时候” 这样，但是感觉空格太多。如果按照单位一起的规则：“GitHub 免费是在 2019年1月4日 左右的时候” 感觉很不匀称很丑啊。

**Comment by MichaelXoXo** [comment]
哇，赞，我之前也写过一个类似的总结 https://michael728.github.io/2018/11/16/habit-writing-reading-working/  但是没有博主总结的全面

**Comment by Jayzongzer** [comment]
！受教了

**Comment by Deoncn** [comment]
受教了！

**Comment by feng528** [comment]
> @yujiangshui
> 日期这个有建议吗？比如：“GitHub 免费是在2019年1月4日左右的时候” 要怎么空？我是：“GitHub 免费是在 2019 年 1 月 4 日左右的时候” 这样，但是感觉空格太多。如果按照单位一起的规则：“GitHub 免费是在 2019年1月4日 左右的时候” 感觉很不匀称很丑啊。

可以 “GitHub 免费是在 2019年01月04日 左右的时候” 这样吗？

**Comment by yujiangshui** [comment]
> 

不知道，我现在用“GitHub 免费是在 2019 年 1 月 4 日左右的时候”
